sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/XMLHighlighting.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SonarQube XML Plugin
+ * Copyright (C) 2010 SonarSource
+ * sonarqube@googlegroups.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sonar.plugins.xml;
+
+import org.sonar.plugins.xml.checks.XmlSourceCode;
+import org.sonar.xml.HighlightingData;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class XMLHighlighting {
+
+
+
+  public static List<HighlightingData> getHighlightingData(XmlSourceCode sourceCode) {
+    List<HighlightingData> highlighting = new ArrayList<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "highlighting". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1488)